AAP-16022 - updated the information to state admin is default userid (#927)
ansible/aap-docs: downstream/modules/platform/con-install-scenario-recommendations.adoc

= Inventory file recommendations based on installation scenarios

[role="_abstract"]
Before selecting your installation method for {PlatformNameShort}, review the following recommendations. Familiarity with these recommendations will streamline the installation process.

* For {PlatformName} or {HubName}: Add an {HubName} host in the `[automationhub]` group.
* For internal databases: `[database]` cannot be used to point to another host in the {PlatformNameShort} cluster.
The database host set to be installed needs to be a unique host.
* Do not install {ControllerName} and {HubName} on the same node for versions of {PlatformNameShort} in a production or customer environment.
This can cause contention issues and heavy resource use.
* Provide a reachable IP address or fully qualified domain name (FQDN) for the `[automationhub]` and `[automationcontroller]` hosts to ensure users can sync and install content from {HubName} from a different node.
Do not use 'localhost'.

> * `admin` is the default userid for the initial log in to Ansible Automation Platform and can not be changed in the inventory file.

Reviewer: ```suggestion
* `admin` is the default userid for the initial log in to Ansible Automation Platform and cannot be changed in the inventory file.
```
